test/collection/list_mutable_extensions_test.dart

```diff
@@ -153,4 +156,16 @@ void testList(
       expect(e.message, allOf(contains("null"), contains("comparator")));
     });
   });
+
+  group("shuffle", () {
+    test("shuffle shuffles items in a list with provided Random object", () {
+      final firstList = mutableListOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+      firstList.shuffle(Random(1));
+      final secondList = mutableListOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+      secondList.shuffle(Random(2));
+      expect(
+          const ListEquality().equals(firstList.asList(), secondList.asList()),
```

Comment:
You can compare `firstList` and `secondList` directly without the ListEquality helper. That's one of the core features of ktdart collections :wink:

```dart
expect(firstList, isNot(equals(secondList)));
```

(Don't forget to remove the `collection` dependency)

Reply:
Great to know!